demos/jobs_for_aws/jobs_demo.c

```diff
@@ -214,6 +214,19 @@
  */
 #define MAKE_STATUS_REPORT( x )    "{\"status\":\"" x "\"}"
 
+/**
+ * @brief The maximum number of times to run the loop in this demo.
+ */
+#ifndef JOBS_MAX_DEMO_COUNT
+    #define JOBS_MAX_DEMO_COUNT    ( 3 )
+#endif
+
+/**
+ * @brief Time in ticks to wait between each cycle of the demo implemented
+ * by RunJobsDemo(), in case a retry is required.
```

Comment:
```suggestion
 * @brief Time in ticks to wait between each iteration of the demo execution, 
 * in case a retry is required from demo execution failure.
```